community/team.html.haml

```diff
@@ -86,3 +86,27 @@ title: Hibernate Team
       $( '#list' ).fadeIn();
     });
   });
+
+%h2{:id => "contributors"} Contributors
+
+.ui.icon.message
+  %i.icon.users
+  .content
+    Hibernate projects are not made just by members,
+    they're the result of the work of hundreds of contributors!
+    <br>
+    Follow the links below to find them all.
+
+.ui.labels.blue
+  %a.ui.label(href="/community/contributors/orm/")
+    Hibernate ORM
+  %a.ui.label(href="/community/contributors/search/")
+    Hibernate Search
+  %a.ui.label(href="/community/contributors/validator/")
+    Hibernate Validator
+  %a.ui.label(href="/community/contributors/reactive/")
+    Hibernate Reactive
+  %a.ui.label(href="/community/contributors/tools/")
+    Hibernate Tools
+  %a.ui.label(href="/community/contributors/ogm/")
+    Hibernate OGM
```

Comment:
@Sanne I added this section at the bottom of the "Team" page to make sure we don't dismiss contributors.